assets/pulpcore-worker.prep

```diff
@@ -2,14 +2,25 @@
 
 ifte
 {
+  foreground
+  {
+    touch
+      /etc/services.d/pulpcore-resource-manager/down
+      /etc/services.d/pulpcore-worker@1/down
+      /etc/services.d/pulpcore-worker@2/down
+  }
+  if
+  {
+    grep -q -e "USE_NEW_WORKER_TYPE\\s*=\\s*True" /etc/pulp/settings.py
+  }
   touch
-    /etc/services.d/pulpcore-resource-manager/down
-    /etc/services.d/pulpcore-worker@1/down
-    /etc/services.d/pulpcore-worker@2/down
+    /etc/services.d/new-pulpcore-resource-manager/down
+    /etc/services.d/redis/down
 }
 {
   touch
+    /etc/services.d/new-pulpcore-resource-manager/down
     /etc/services.d/new-pulpcore-worker@1/down
     /etc/services.d/new-pulpcore-worker@2/down
 }
-grep -q -e "USE_NEW_WORKER_TYPE\\s*=\\s*True" /etc/pulp/settings.py
+python3 -c "import sys; from packaging.version import parse; from pulpcore.app.apps import PulpAppConfig; sys.exit(0 if parse(PulpAppConfig.version) >= parse('3.13.0.dev0') else 1)"
```

Comment:
sweet!

Reply:
well, it's a mouthful, but i found no better way.